cognite/seismic/protos/v1/seismic_service.proto

```diff
@@ -107,4 +107,6 @@ service SeismicAPI {
     Can retrieve a full file or create a new cropped file filtering on areas of interest only.
     **/
     rpc GetSegYFile (SegYSeismicRequest) returns (stream SegYSeismicResponse) {}
+
+    rpc SearchFiles(SearchFilesRequest) returns (stream File) {}
```

Comment:
Oh no we forgot a doc comment...

Reply:
Oooh, I see another problem. Looking at the v0 GetFile endpoint, there's a GetFileResponse message wrapping File protobufs. All the custom headers and stuff are in the GetFileResponse protobuf. This is... unfortunate.

But now that we're redoing the API in v1 we can fix that!  :grinning: